SpatialGDK/Source/SpatialGDK/Public/Schema/Interest.h

```diff
@@ -54,7 +54,7 @@ struct QueryConstraint
 	TSchemaOption<uint32> ComponentConstraint;
 	TArray<QueryConstraint> AndConstraint;
 	TArray<QueryConstraint> OrConstraint;
-	// TSchemaOption<> SelfConstraint; TODO
+	TSchemaOption<bool> SelfConstraint;
```

Comment:
why an optional bool? Could it be a sort of void optional, or just a bool instead? I'm not sure what it means for this to be set but false.

Reply:
Setting it to true or false would set it, but I see that's confusing. Can't have a option<void> so will just make it a bool.